cores/esp8266/Schedule.cpp

```diff
@@ -60,19 +75,45 @@ bool schedule_function(std::function<void(void)> fn)
         sLast->mNext = item;
     }
     sLast = item;
+
+    if (repeat_us)
+        item->callNow.reset(repeat_us);
+
     return true;
 }
 
+IRAM_ATTR // called from ISR
+bool schedule_function(std::function<void(void)> fn)
+{
+    return schedule_function_us([&fn](){ fn(); return false; }, 0);
+}
+
 void run_scheduled_functions()
```

Comment:
This function should have a big phat comment explaining the assumption that ISRs will ever only add to the list and never remove items.